portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -64,4 +78,43 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String json = new Gson().toJson(commentList);
     response.getWriter().println(json);
   }
+
+    private String getUsername(String id) {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Query query =
+            new Query("user")
+                .setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
+        PreparedQuery results = datastore.prepare(query);
+        Entity entity = results.asSingleEntity();
+
+        if (entity == null) {
+            return null;
+        }
+
+        System.out.println("======NOT+NULLLLLLL=======");
+
+
+        String nickname = (String) entity.getProperty("username");
+        return nickname;
+  }
+
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    boolean deleteAll = true;
+    if (deleteAll) {
+        Query query = new Query("Comment");
+
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = datastore.prepare(query);
+
+    List<Key> commentKeys = new ArrayList<>();
```

Comment:
Formatting: indent this consistently with the other lines.

Reply:
sps clang format